examples/surrounding_nitrogen_bonds/nn_bond_length_histogram.py

```diff
@@ -0,0 +1,108 @@
+"""
+Script for generating a histogram based on the bond length of
+single nitrogen-nitrogen bonds
+Usage:
+    python nn_bond_length_histogram.py --smiles_database [DATABASE] --fingerprinted [T or F]
+"""
+
+from openeye import oechem, oeomega, oequacpac, oeff
+import argparse
+import matplotlib.pyplot as plt
+import math
+
+def generateHistogram():
```

Comment:
Please use `snake_case` -- I try to follow the Google style guide (specifically this section: https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations) in DANCE. This applies to the other functions too.

Reply:
this would apply to all variables names also, correct?